pages/docs/guidelines/react.mdx

```diff
@@ -0,0 +1,473 @@
+
+### API Call
+
+The majority of web applications developed are full stack applications so, it needs a reliable tool to handle all of the API calls in the frontend part. So, we use "Axios" for all the API calls here at Truemark. Axios is an HTTP client which handles all of the HTTP requests sent from the frontend. It has a lot of features that make any HTTP call easy to handle.
+
+axios can be added to any project using npm or yarn.
+
+Installing axios using `npm`
+
+```
+npm install axios
+```
+
+Installing axios using `yarn`
+
+```
+yarn add axios
+```
+
+Find the npm package and github link below:
+
+NPM Package: [axios npm package](https://www.npmjs.com/package/axios)
+
+GitHub Repository: [axios github repo](https://github.com/axios/axios)
+
+### Drag and drop
+
+While developing a web application with drag and drop feature, it is always a challenge for getting the output as expected. Drag and drop functionality has always been one of the complex functionalities to handle so, we use an extremely easy tool — "React DnD" for all of our draggable stuff in React app.
+
+React DnD can be added to any project using npm or yarn.
+
+Installing React DnD using `npm`
+
+```
+npm install react-dnd
+```
+
+Installing React DnD using `yarn`
+
+```
+yarn add react-dnd
+```
+
+Find the npm package and github link below:
+
+NPM Package: [react-dnd npm package](https://www.npmjs.com/package/react-dnd)
+
+GitHub Repository: [react-dnd github repo](https://github.com/react-dnd/react-dnd)
+
+### CSS Preprocessor
+
+We use SASS as our CSS preprocessor here at Truemark. Having more features and structure than the regular CSS, SASS tops our list so, we use SASS for all of our frontend projects at Truemark. You can also visit the SCSS guidelines provided in Truemark's guideline docs.
+
+SASS can be added to any project using npm or yarn.
+
+Installing SASS using `npm`
+
+```
+npm install sass
+```
+
+Installing SASS using `yarn`
+
+```
+yarn add sass
+```
+
+Find the npm package and github link below:
+
+NPM Package: [SASS npm package](https://www.npmjs.com/package/sass)
+
+GitHub Repository: [SASS github repo](https://github.com/sass/dart-sass)
+
+### SVG Icons
+
+Icons are a great way to convey the message in any frontend application and every frontend application's presentation and design are uplifted by the use of icons. Therefore, we use the "react-fontawesome" icon package for all of our icons at Truemark.
+
+react-fontawesome can be added to any project using npm or yarn.
+
+Installing react-fontawesome using `npm`
+
+```
+npm i @fortawesome/react-fontawesome
+```
+
+Installing react-fontawesome using `yarn`
+
+```
+yarn add @fortawesome/react-fontawesome
+```
+
+Find the npm package and github link below:
+
+NPM Package: [react-fontawesome npm package](https://www.npmjs.com/package/@fortawesome/react-fontawesome)
+
+GitHub Repository: [react-fontawesome github repo](https://github.com/FortAwesome/react-fontawesome)
```

Comment:
I think we are missing official documentation site. 
example: `https://fontawesome.com/`
Let's update for lib where we have official documentation site